src/components/ModelingMachineProvider.tsx

```diff
@@ -800,19 +800,26 @@ export const ModelingMachineProvider = ({
         }),
         'animate-to-sketch': fromPromise(
           async ({ input: { selectionRanges } }) => {
+            const artifact = selectionRanges.graphSelections[0].artifact
             const plane = getPlaneFromArtifact(
-              selectionRanges.graphSelections[0].artifact,
+              artifact,
               engineCommandManager.artifactGraph
             )
             if (err(plane)) return Promise.reject(plane)
+            // if the user selected a segment, make sure we enter the right sketch as there can be multiple on a plan
```

Comment:
```suggestion
            // if the user selected a segment, make sure we enter the right sketch as there can be multiple on a plane
```

Reply:
As discussed on Slack, this will be for another time :)

Reply:
I fixed this typo in another branch since it missed the ship sailing, it'll get in :)